web/components/input/buttons/SecondaryButton/SecondaryButton.sass

```diff
@@ -0,0 +1,13 @@
+button.secondary
+  background: $gradient-secondary
```

Comment:
We should make the styling for `SecondaryButton` the same as `PrimaryButton` either in this PR or create a follow-up issue. 

OR we could refactor `PrimaryButton` and `SecondaryButton` into one component as I've already mentioned

Reply:
Ok we've decided not to refactor it, but i'd still like to see consistent `sass` between the two components? Like I'm not seeing the `@include gradient-hover-animation($gradient-primary, $gradient-primary-highlight)` piece here

Reply:
What do you mean about the styling? I might agree on making them the same with only colour variations, but the special CSS required to make `linear-gradient` transition properly sorta makes that more effort than its worth IMO 😞

Reply:
> Like I'm not seeing the @include gradient-hover-animation($gradient-primary, $gradient-primary-highlight) piece here

The secondary button transitions to a flat color `$secondary-highlight`, which works with `-webkit-transition`, but the primary button's `gradient`-to-`gradient` transition doesn't rip, hence the need for the mixin on one and not the other.

Reply:
A lot of the shared styles are already in `index.sass` of `buttons`, which worked pretty well since they pretty much all applied to the progress group buttons as well (see the CSS in #21)

Reply:
Thought: we should be careful about making the styling too tightly coupled between the components, because it makes us really inflexible when changes are requested - it's easier to copy styles across multiple components than it is to disentangle a property from tightly coupled styles.

Reply:
Okay nevermind, i see what you mean

and yeah i'm a noob to css so i don't fully understand the pros/cons of the two approaches but i think if we have a fully fleshed out style guide then it's possible to achieve styles that are tightly coupled between the components

i guess let's bring it up in our channel/next meeting?

Reply:
🚀